Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="70" judge-id="27" status="SE" scoring="ACM" run-tests="1" contest-id="3205" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>9d69718d-78a5-4fc7-90e0-fc5960e1d7b9</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="283" real-time="309" max-memory-used="18698240" >
  <program-stats-str>{ utime=227, stime=56, ptime=283, rtime=309, maxvsz=18698240, maxrss=19443712, nvcsw=1, nivcsw=53 }</program-stats-str>
      <input size="10">1 2 3 2 1
</input>
      <output size="0"></output>
      <stderr size="274">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000070&quot;, line 8, in &lt;module&gt;
    if (s0):
NameError: name 's0' is not defined
</stderr>
    </test>
  </tests>
</testing-report>
